### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by ntimanu**
In this project we:

- Added desktop pages:

1. Info page

2. Work pages

3. About page

4. Contact form

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -713,289 +665,42 @@ ul li {
...
-    height: 924px;
-    left: 0;
-    top: 3610px;
-    background: #fff;
-    border-radius: 0 10% 0 0/0 10% 0 0;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 4.8rem;
+    padding: 2.8rem;
   }
```
Comment (Qoosim) [inline style.css]: - [x] The about myself takes 40% of the width while the language part takes 60% with margin between them. Your design does not look the exact way as we have it on the figma design. Kindly look into that.

### Your Design
your-design-about

### Figma Design
temp-design-about


- [x] The framework and skill underlines span across the whole 60% width, not only the width of the text. Kindly fix that too.

### Your Design
your-underline

### Figma Design
temp-underline

File: style.css [hunk 2]
```diff
@@ -1066,219 +733,48 @@ ul li {
...
-    width: 447px;
-    margin-left: auto;
-    margin-right: auto;
-    top: 420px;
-    bottom: 0;
-    background: #fff;
-    border: 1px solid #cfd8dc;
-    border-radius: 2px;
+  .contactme > p {
+    margin: auto 15%;
```
Comment (Qoosim) [inline style.css]: - [x] Kindly reduce the width of your form paragraphy in order to match the figma design. Take a look :point_down: 

###  Your Design
your-form

### Figma Design
temp-form

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @ntimanu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (ntimanu) [reply style.css]: Done

File: style.css (hunk 2 above)
Comment (ntimanu) [reply style.css]: Done

**Review by TadesseDev — APPROVED** [review]
### Hi @ntimanu 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**